### PR TITLE
this includes various fixes

### PR DESCRIPTION
**Pull request by laskasn**
- Increased the AUP width, the wayf listing width and removed the logo extra padding.

### CODE REVIEW

**Comment by cgeorgilakis** [comment]
https://jira.argo.grnet.gr/browse/RCIAM-1124 ( all changes related to our theme)

File: src/main/resources/theme/rciam/login/login-oauth-grant.ftl [hunk 1]
```diff
@@ -13,16 +13,18 @@
         </p>
     <#elseif section = "form">
         <div id="kc-oauth" class="content-area">
-            <h3>${msg("oauthGrantRequest")}</h3>
-            <ul>
-                <#if oauth.clientScopesRequested??>
```
Comment (laskasn) [inline src/main/resources/theme/rciam/login/login-oauth-grant.ftl:18]: I actually moved this #if one level up, to include the message...
I don't know if you agree that if the list is empty, it's better not to show a message 
"This service would like to have acccess to:"

...with nothing below...

Comment (cgeorgilakis) [reply src/main/resources/theme/rciam/login/login-oauth-grant.ftl:18]: This is the situation for a client with consent required, Display Client On Consent Screen off and without scopes with consent.
I am not sure what is best for such a situation. Maybe another message?

File: src/main/resources/theme/rciam/login/messages/messages_en.properties [hunk 2]
```diff
@@ -1,5 +1,5 @@
 loginAccountTitle=Choose your academic/social account
-oauthGrantRequest=This service would like to \:
+oauthGrantRequest=This service would like to have access to the following \:
```
Comment (cgeorgilakis) [inline src/main/resources/theme/rciam/login/messages/messages_en.properties]: I disagree with this change.
See the ggus screenshot for example.

File: src/main/resources/theme/rciam/login/login-oauth-grant.ftl (hunk 1 above)
Comment (laskasn) [reply src/main/resources/theme/rciam/login/login-oauth-grant.ftl:18]: Indeed, displaying a different message if empty list, would be the best!

Comment (laskasn) [reply src/main/resources/theme/rciam/login/login-oauth-grant.ftl:18]: ok, let's change that

**Pull request merged**